projects/plutus/test/test_calibration.c

```diff
@@ -0,0 +1,64 @@
+#include <inttypes.h>
+
+#include "current_calibration.h"
+#include "current_sense.h"
+#include "delay.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#define TEST_MAX_CURRENT_POINT 3
```

Comment:
What is this used for?

Reply:
Since the module needs to know what current the adc readings correspond to beforehand, I kept an arbitrary max point that can be used during testing.

Reply:
Please fix your naming and add documentation.

Reply:
Fixed